<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>nameDialog</class>
 <widget class="QDialog" name="nameDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>110</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Set pixel transformation</string>
  </property>
  <widget class="QLabel" name="origLabel">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>10</y>
     <width>160</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Object Name&lt;/span&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>75</x>
     <y>75</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>40</y>
     <width>220</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>okButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
